subsys/net/lib/download_client/src/coap.c

```diff
@@ -126,12 +129,15 @@ int coap_request_send(struct download_client *client)
 		return err;
 	}
 
-	err = coap_packet_append_option(&request, COAP_OPTION_URI_PATH,
-					file, strlen(file));
-	if (err) {
-		LOG_ERR("Unable add option to request");
-		return err;
-	}
+	path_elem = strtok_r(file, COAP_PATH_ELEM_DELIM, &path_elem_saveptr);
+	do {
+		err = coap_packet_append_option(&request, COAP_OPTION_URI_PATH,
+		                                path_elem, strlen(path_elem));
```

Comment:
Please use tabs for indention when possible.

Reply:
It was indented with tabs (and aligned with spaces), same as in same file line 101.
Anyway, changed it to a single tab as most of the reset of the file uses.

Reply:
You can still align with spaces, the point is to use as many tabs as possible before using spaces (so the number of spaces used for alignment is < 8) - this is the requirement we've adopted from upstream Zephyr and is verified by the CI (otherwise I wouldn't even notice I guess).

Reply:
Want it aligned or are you fine with the current 1-tab indentation? I don't care.